lib/src/pdfjs_global.dart

```diff
@@ -30,26 +30,26 @@ enum VERBOSITY_LEVELS {
 
 class PDFJS {
   /// the local copy of versionSafePdfJsContext once initialized
-  static JsObject _pdfJsContext;
+  static JsObject? _pdfJsContext;
 
   /// the local copy of versionSafePdfJsViewerContext once initialized
-  static JsObject _pdfJsViewerContext;
+  static JsObject? _pdfJsViewerContext;
 
   /// Getter for obtaining the pdfjs context for various versions
   /// of pdf.js
-  static JsObject get versionSafePdfJsContext {
+  static JsObject? get versionSafePdfJsContext {
     if(_pdfJsContext != null) {
       return _pdfJsContext;
     }
     
     // ^v2.5.207
     if(context['pdfjsLib'] != null) {
-      _pdfJsContext = context['pdfjsLib'] as JsObject;
+      _pdfJsContext = context['pdfjsLib'] as JsObject?;
```

Comment:
this null check is already done, can this be changed? Would save a lot of guessing for this getter to return a non-null value
```suggestion
      _pdfJsContext = context['pdfjsLib'] as JsObject!;
```

Reply:
As per the suggestion, I have tried it in local. It is giving me error: `Expected to find ';'.`